ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -563,6 +563,54 @@
 | L3 Gateway Inter-domain | True |
 {%         endif %}
 {%     endif %}
+{%     if router_bgp.address_family_link_state is arista.avd.defined %}
+
+#### Router BGP Link-State Address Family
+{%         if router_bgp.address_family_link_state.neighbors is arista.avd.defined %}
+
+##### Link-State Neighbors
+
+| Neighbor | Activate | Missing policy In action | Missing policy Out action |
+| -------- | -------- | ------------------------ | ------------------------- |
+{%             for neighbor in router_bgp.address_family_link_state.neighbors | arista.avd.natural_sort('ip_address') %}
+{%                 set missing_policy_in = neighbor.missing_policy.direction_in_action | arista.avd.default("-") %}
+{%                 set missing_policy_out = neighbor.missing_policy.direction_out_action | arista.avd.default("-") %}
+| {{ neighbor.ip_address }} | {{ neighbor.activate | arista.avd.default(false) }} | {{ missing_policy_in }} | {{ missing_policy_out }} |
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.address_family_link_state.peer_groups is arista.avd.defined %}
+
+##### Link-State Peer Groups
+
+| Peer Group | Activate | Missing policy In action | Missing policy Out action |
+| ---------- | -------- | ------------------------ | ------------------------- |
+{%             for peer_group in router_bgp.address_family_link_state.peer_groups | arista.avd.natural_sort('name') %}
+{%                 set missing_policy_in = peer_group.missing_policy.direction_in_action | arista.avd.default("-") %}
+{%                 set missing_policy_out = peer_group.missing_policy.direction_out_action | arista.avd.default("-") %}
+| {{ peer_group.name }} | {{ peer_group.activate | arista.avd.default(false) }} | {{ missing_policy_in }} | {{ missing_policy_out }} |
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.address_family_link_state.path_selection is arista.avd.defined %}
+
+##### Link-State Path Selection configuration
+
+| Settings | Value |
+| -------- | ----- |
+{%             if router_bgp.address_family_link_state.path_selection.roles is arista.avd.defined %}
+{%                 set path_selection_roles = [] %}
+{%                 if router_bgp.address_family_link_state.path_selection.roles.producer is arista.avd.defined(true) %}
+{%                     do path_selection_roles.append("*  producer") %}
```

Comment:
```suggestion
{%                     do path_selection_roles.append("* producer") %}
```

Reply:
Normally we do not use bullet lists here but just dash.

Reply:
[c15b6fc](https://github.com/aristanetworks/ansible-avd/pull/3211/commits/c15b6fc3d0da3a11bedc0b807f41fe19bcc97bdb)